### DISCUSSION TITLE
Is there a reason TipTap deviated from Prosemirror in node naming?

### DISCUSSION THREAD

**Discussion by philiprenich** [ueberdosis/tiptap#5619] [General Questions] [answered]
I'm working on porting over a DOM serializer to C# and have run into the annoying realisation that when exporting to JSON, TipTap uses camelCased naming convention for nodes and marks: `listItem`, `codeBlock` etc. Prosemirror uses `list_item` and `code_block`

I'm working alongside a C# port of Prosemirror so am trying to follow those conventions. It's definitely surmountable to deal with the difference, but more custom code to handle it and I was curious what the reasoning was for this change.

Similarily, Tiptap uses `italic` and `bold` whereas Prosemirror uses `em` and `strong`

I'm not looking for it to be changed (that's Breaking, and a major version bump), but to make sure there isn't a reason that will impact my code I should be aware of. Thanks!

**Comment by nperez0111** [comment, marked as answer]
The naming of extensions was decided long ago. The names should not affect your application in anyway and you can override extension names
